packages/webapp/src/stories/Pages/LocationDetails/AreaDetails/AreaDetails.stories.jsx

```diff
@@ -55,6 +57,7 @@ Post.args = {
   setError: (data) => {},
   control: (data) => {},
   history: (data) => {},
+  match: { params: { location_id: 1 } },
```

Comment:
watch?

Reply:
match is from react router
https://v5.reactrouter.com/web/api/match. 
It says location id in the URL is 1